src/custom/components/OrdersPanel/index.tsx

```diff
@@ -0,0 +1,420 @@
+import { AbstractConnector } from '@web3-react/abstract-connector'
```

Comment:
Just realized this is a mod file, but you forgot to name it `OrdersPanelMod`

Reply:
Actually not a mod, but a complete new component :)

Reply:
:open_mouth: